lib/inferno_requirements_tools/tasks/collect_requirements.rb

```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
+
+require 'CSV'
+require 'roo'
+
+module InfernoRequirementsTools
+  module Tasks
+    # This class manages the collection of requirements details from
+    # requirements planning excel workbooks into a CSV representation.
+    # Currently splits out Requirements and Planned Not Tested Requirements
+    # into two separate files.
+    #
+    # The `run_check` method will check whether the previously generated file is up-to-date.
+    class CollectRequirements
+      # Update these constants based on the test kit.
+      TEST_KIT_ID = 'carin-for-blue-button-test-kit'
+      INPUT_SETS = ['hl7.fhir.us.carin-bb_2.0.0'].freeze
```

Comment:
I expect more input sets to be added in the future, e.g., around some inherited US Core requirements, but this is fine to start with.